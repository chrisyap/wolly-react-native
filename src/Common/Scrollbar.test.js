import React from 'react';
import ScrollBar from './Scrollbar';

describe('Scrollbar', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ScrollBar />);
    expect(wrapper).toMatchSnapshot();
  });
});
