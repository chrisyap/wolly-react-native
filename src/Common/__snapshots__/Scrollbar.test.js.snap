// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scrollbar renders correctly 1`] = `
<Styled(View)
  height="100%"
  position="absolute"
  py="5px"
>
  <Styled(Component)
    backgroundColor="black"
    borderRadius={8}
    style={
      Object {
        "opacity": undefined,
        "transform": Array [
          Object {
            "translateY": undefined,
          },
        ],
      }
    }
  />
</Styled(View)>
`;
